tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/OpenFOAM/0.2/T

```diff
@@ -1,9 +1,9 @@
 /*--------------------------------*- C++ -*----------------------------------*\
-| =========                 |                                                 |
-| \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox           |
-|  \\    /   O peration     | Version:  4.1                                   |
-|   \\  /    A nd           | Web:      www.OpenFOAM.org                      |
-|    \\/     M anipulation  |                                                 |
+  =========                 |
+  \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox
+   \\    /   O peration     | Website:  https://openfoam.org
+    \\  /    A nd           | Version:  6
```

Comment:
This test uses the APT package `openfoam5`, right? Why do the headers show 6?

Reply:
Good point. The machine where I generated the reference results uses OF6. I will rerun the examples with OF5.

Reply:
Apart from the different headers, I also see minor differences in the results, which should not be a problem, since we are now comparing the results with some tolerance. But if you have time, it would be nice to rerun with the same OpenFOAM version. :+1:

Reply:
I rerun the experiments and added new reference results in https://github.com/precice/systemtests/pull/132/commits/93e79b3ae57e1773e06561b2de707c7305087314. I hope this resolves this point.

Reply:
Regarding the minor differences: Since we do not compare with an analytical solution or experimental results as far as I understand minor changes are ok, as long as the results still "look" good. I checked the results in paraview and could not find any problems.